mod-orders/examples/composite_purchase_orders.sample

```diff
@@ -68,7 +70,9 @@
             "quantityPhysical": 2,
             "poLineEstimatedPrice": 28.00
           },
-          "orderFormat": "Physical Resource"
+          "isPackage": true,
+          "orderFormat": "Physical Resource",
```

Comment:
same here, please add source

Reply:
@KVupp ,
this came from MODORDERS-346 and will be done in other PR because I reverted merging of MODORDERS-346.